gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-bin.zip
```

Comment:
we can also run the command `./gradlew wrapper --gradle-version=8.13` so that we get the updated gradle wrapper commited as well.